### PR TITLE
docs(rulesets): link to spectral-rulesets

### PR DESCRIPTION
**Pull request by philsturgeon**
Some rebase went wrong and my previous attempt at doing this (https://github.com/stoplightio/spectral/commit/3ad596b52e75f90c00bfb961350f6096f9a902a0) fell into the void.

### CODE REVIEW

**Review by heitortsergent — DISMISSED** [review]
lgtm!

File: README.md [hunk 1]
```diff
@@ -75,19 +77,18 @@ If you need help using Spectral or have any questions, please use [GitHub Discus
...
+- [APIs You Won't Hate](https://github.com/apisyouwonthate/style-guide) - An opinionated collection of rules based on advice in the [APIs You Won't Hate](https://apisyouwonthate.com/) community.
+- [Azure](https://github.com/Azure/azure-api-style-guide/blob/main/spectral.yaml) - Ruleset and complimentary style guide for creating OpenAPI 2 or 3 definitions of Azure services.
+- [Box](https://github.com/box/box-openapi/blob/main/.spectral.yml) - Lots of [Custom Functions](https://meta.stoplight.io/docs/spectral/ZG9jOjI1MTkw-custom-functions) being used to enforce good practices that the Box API governance folks are interested in.
+- [DigitalOcean](https://github.com/digitalocean/openapi/blob/main/spectral/ruleset.yml) - Keeping their OpenAPI nice and tidy, enforcing use of `$ref` (probably to minimize conflicts), naming conventions for Operation IDs, and all sorts of other handy OpenAPI tips.
+- [Tranascom](https://github.com/transcom/mymove/blob/master/swagger-def/.spectral.yml) - Don't even think about using anything other than `application/json`.
 
-### I want to lint my OpenAPI documents but don't want to implement Spectral right now
```
Comment (mnaumanali94) [inline README.md:84]: Why remove this? @heitortsergent

Comment (heitortsergent) [reply README.md:84]: I'm leaning on @philsturgeon experience here in removing the other two questions, for this one specifically we're also linking to [Stoplight Studio on line 97](https://github.com/stoplightio/spectral/pull/2234/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R97) so I think it's ok to remove it from here. But just my two cents!

Comment (mnaumanali94) [reply README.md:84]: I agree with removing the other two!

I think this is a good place to direct people to the platform implementation who don't want the heavy lifting needed to setup spectral. Thoughts on that? @philsturgeon

Comment (philsturgeon) [reply README.md:84]: The goal is to get people over to platform who might be interested but the FAQ is kinda awkward way to do it. The other rules don’t seem relevant anymore so a one link FAQ doesn’t seem useful.

**Review by mnaumanali94 — APPROVED** [review]

**Pull request merged**